forui/test/src/widgets/popover/popover_test.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:forui/forui.dart';
+import '../../test_scaffold.dart';
+
+void main() {
+  group('FPopover', () {
+    testWidgets('tap outside hides popover', (tester) async {
+      final controller = FPopoverController(vsync: const TestVSync());
+
+      await tester.pumpWidget(
+        MaterialApp(
```

Comment:
Should we remove `MaterialApp`? To ensure it works stand alone.

Reply:
It'll be addressed in the tooltip PR